### PR TITLE
Usercard search

### PR DESCRIPTION
**Pull request by nanaya**
Now with card instead of just avatar.

Also fix alignment stuff.

### CODE REVIEW

File: resources/views/objects/_usercard.blade.php [hunk 1]
```diff
@@ -34,22 +47,26 @@
                     <div class="usercard__username">{{isset($user) ? $user->username : 'Loading...'}}</div>
                     <div class="usercard__flags">
                         @if (isset($loading))
-                            @include('objects._country_flag', ['country_code' => 'XX'])
+                            <div class="usercard__flag">
```
Comment (nekodex) [inline resources/views/objects/_usercard.blade.php]: move this div to outside of `@if`?

File: resources/views/objects/_usercard.blade.php [hunk 2]
```diff
@@ -34,22 +47,26 @@
...
-                                <div class="js-react--friendButton" data-target="{{$user->user_id}}"></div>
+                            <div class="usercard__flag">
+                                @if (isset($user->country))
+                                    @include('objects._country_flag', [
+                                        'country_code' => $user->country->acronym,
+                                        'country_name' => $user->country->name,
+                                    ])
+                                @endif
+                            </div>
+                            <div class="usercard__flag">
```
Comment (nekodex) [inline resources/views/objects/_usercard.blade.php]: wait, wut ಠ_ಠ

**Comment by nekodex** [comment]
icon alignment in the friend button seems off now (uncentered?)

before:

after:

**Comment by nanaya** [comment]
But I didn't change the button styling at all ಠ_ಠ

**Comment by nanaya** [comment]
Apparently because it was just accidentally centered, not properly centered.

**Comment by nekodex** [comment]
the icon alignment during hover is still misaligned..

**Comment by nanaya** [comment]
which icon?

**Comment by nekodex** [comment]
friend button icon

**Pull request merged**